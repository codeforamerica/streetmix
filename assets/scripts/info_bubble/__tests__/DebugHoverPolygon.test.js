/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import { DebugHoverPolygon } from '../DebugHoverPolygon'

describe('DebugHoverPolygon', () => {
  it('renders if enabled', () => {
    const wrapper = shallow(<DebugHoverPolygon enabled hoverPolygon={[]} />)
    expect(wrapper).toMatchSnapshot()
  })

  it('renders nothing if disabled', () => {
    const wrapper = shallow(<DebugHoverPolygon enabled={false} />)
    expect(wrapper).toMatchSnapshot()
  })
})
