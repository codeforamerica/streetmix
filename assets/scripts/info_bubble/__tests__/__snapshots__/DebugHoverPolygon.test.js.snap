// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DebugHoverPolygon renders if enabled 1`] = `
<div
  className="debug-hover-polygon"
>
  <canvas
    height={768}
    width={1024}
  />
</div>
`;

exports[`DebugHoverPolygon renders nothing if disabled 1`] = `""`;
